v1.1.0 is broken in go1.16

Hi,

Thank you for this awesome project.

I'm getting the following error when importing the package in Go 1.16

```bash
../../go/pkg/mod/github.com/twpayne/go-polyline@v1.1.0/polyline.go:93:12: undefined: math.MaxUint
../../go/pkg/mod/github.com/twpayne/go-polyline@v1.1.0/polyline.go:94:10: undefined: math.MinInt
```
It seems there are some breaking changes in math package in recent Go versions.
https://go-review.googlesource.com/c/go/+/317911/

Please let me know if you want any contribution from my end.

Thank you again.

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -12,6 +12,7 @@ jobs:
     strategy:
       matrix:
         go-version:
+          - 1.16.x
           - 1.17.x
     steps:
     - name: Set up Go

--- a/export_test.go
+++ b/export_test.go
@@ -1,0 +1,6 @@
+package polyline
+
+const (
+	MaxInt = maxInt
+	MinInt = minInt
+)

--- a/polyline.go
+++ b/polyline.go
@@ -90,8 +90,8 @@ func DecodeInt(buf []byte) (int, []byte, error) {
 		return 0, nil, err
 	case u&1 == 0:
 		return int(u >> 1), buf, nil
-	case u == math.MaxUint:
-		return math.MinInt, buf, nil
+	case u == maxUint:
+		return minInt, buf, nil
 	default:
 		return -int((u + 1) >> 1), buf, nil
 	}

--- a/polyline_go116.go
+++ b/polyline_go116.go
@@ -1,0 +1,12 @@
+//go:build !go1.17
+// +build !go1.17
+
+package polyline
+
+import "math"
+
+const (
+	maxInt  = math.MaxInt64
+	maxUint = math.MaxUint64
+	minInt  = math.MinInt64
+)

--- a/polyline_go117.go
+++ b/polyline_go117.go
@@ -1,0 +1,12 @@
+//go:build go1.17
+// +build go1.17
+
+package polyline
+
+import "math"
+
+const (
+	maxUint = math.MaxUint
+	maxInt  = math.MaxInt
+	minInt  = math.MinInt
+)

--- a/polyline_test.go
+++ b/polyline_test.go
@@ -99,10 +99,10 @@ func TestInt(t *testing.T) {
 		{i: -75000, s: "nnqC"},
 		{i: 255200, s: "_mqN"},
 		{i: -550300, s: "vxq`@"},
-		{i: math.MaxInt, s: "}~~~~~~~~~~~N"},
-		{i: math.MaxInt - 1, s: "{~~~~~~~~~~~N"},
-		{i: math.MinInt + 1, s: "|~~~~~~~~~~~N"},
-		{i: math.MinInt, s: "~~~~~~~~~~~~N"},
+		{i: polyline.MaxInt, s: "}~~~~~~~~~~~N"},
+		{i: polyline.MaxInt - 1, s: "{~~~~~~~~~~~N"},
+		{i: polyline.MinInt + 1, s: "|~~~~~~~~~~~N"},
+		{i: polyline.MinInt, s: "~~~~~~~~~~~~N"},
 	} {
 		got, b, err := polyline.DecodeInt([]byte(tc.s))
 		assert.NoError(t, err)

